orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -573,6 +573,51 @@ OpSchema& RegisterLambOpSchema(OpSchema&& op_schema) {
   return op_schema;
 }
 
+bool SoftMaxGradFunctionBodyBuilder(const FunctionBodyBuildContext& ctx, const OpSchema& schema, FunctionProto& functionProto) {
```

Comment:
This function body was written for Softmax-11's gradient. 
Is the function body still valid for Softmax-11's gradient?